src/scripts/components/elements/validation.js

```diff
@@ -66,4 +73,49 @@ export default class Validation {
       'h5p-reader-question-required-wrapper', 'h5p-reader-question-hidden'
     );
   }
+
+  /**
+   * Validate input/editor content
+   * @param {number} contentLength length of content
+   * @returns {boolean}.

```

Comment:
It's good practice to add a description, here e.g. `True, if input validates. Else false.`, so if one created automated documentation from JSDoc, the values would bear meaning.